File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Color.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style
{
    public class Color : ASTNode, IStyle
    {
        public System.Drawing.Color Value { get; }

        private System.Drawing.Color _value;

        public Color(int lineNumber, System.Drawing.Color value)
        {
            _lineNumber = lineNumber;
            _value = value;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}

File: Joanna_Laurence/src/main/java/org/uva/sea/languages/qls/parser/elements/style/Color.java
package org.uva.sea.languages.qls.parser.elements.style;

import org.antlr.v4.runtime.Token;
import org.uva.sea.languages.qls.parser.visitor.IStyleASTVisitor;

public class Color extends StyleSpecification {

    private final String colorCode;

    public Color(final Token token, final String colorCode) {
        super(token);
        this.colorCode = colorCode;
    }

    public final String getColorCode() {
        return this.colorCode;
    }

    @Override
    public final <T> T accept(final IStyleASTVisitor<T> visitor) {
        return visitor.visit(this);
    }
}
